sql database/scripts/createTrain.py

```diff
@@ -11,66 +11,143 @@
 
 # create a connection
 def create_connection(db_file):
-	""" create a db connection to database specified
-		by db_file
-	:param db_file: database file path
-	:return: Connection object or None
-	"""
-	try:
-		conn = sqlite3.connect(db_file);
-		return conn
-	except Error as e:
-		print(e)
-
-	return None
+    """ create a db connection to database specified
+        by db_file
+    :param db_file: database file path
+    :return: Connection object or None
+    """
+    try:
+        conn = sqlite3.connect(db_file);
+        return conn
+    except Error as e:
+        print(e)
+
+    return None
 
 # select all images in the database
 def select_all_images(conn):
-	"""	Query all rows in the images tables
-	:param conn: the Connection object
-	:return: list of rows
-	"""
-	cur = conn.cursor()
-	cur.execute("SELECT Filename FROM images")
-
-	return cur.fetchall()
+    """    Query all rows in the images tables
+    :param conn: the Connection object
+    :return: list of rows
+    """
+    cur = conn.cursor()
+    cur.execute("SELECT Filename FROM images")
+    names = [x[0] for x in cursor.description]
+    return [cur.fetchall(), names]
 
 # select all images with a certain species
 def select_image_by_query(conn, query):
-	"""	Query image by species
-	:param conn: the Connection Object
-	:param query: manual query
-	:return: all rows of the species
-	"""
-	cur = conn.cursor()
-	cur.execute(query)
+    """    Query image by species
+    :param conn: the Connection Object
+    :param query: manual query
+    :return: all rows of the species
+    """
+    cur = conn.cursor()
+    cursor = cur.execute(query)
+    names = [x[0] for x in cursor.description]
+    return [cur.fetchall(), names]
+
+# create train and val csv files given user input
+def create_csv (df,date):
+    df = df.rename({'Filename': 'images', 'GTruth': 'label'}, axis='columns')
```

Comment:
Does there need to be an inconsistency between the column definitions from when it's initialized to the csv?
Also we should be importing this column names as constants from `utils/constants.py`. It could be defined as a class `SQLConstants`. Better practice. Avoid hardcoding.

Reply:
Reuse `'images'` and `'label'` from the `utils/constants.py` as well.